examples/3D-pol-to-birefringence.py

```diff
@@ -78,24 +78,15 @@
 ## Save to zarr
 # Save birefringence
 writer = WaveorderWriter("./output")
-writer.create_zarr_root("birefringence_" + timestamp)
+writer.create_zarr_root("reconstructions_" + timestamp)
```

Comment:
Should we standardize what this zarr store is called?

Reply:
Good idea...I just changed all of the scripts to save to `./output/reconstructsion_<timestamp>`. What do you think of that pattern?

Reply:
I'm not sure. I wish it was a bit more descriptive. Maybe something like `recOrder_recon.zarr` to differentiate it from other data the user may put in that directory e.g. `fluor_decon.zarr`?  I'm still not sure if having two zarr stores in the same folder is allowed by ome standards

Reply:
> I'm still not sure if having two zarr stores in the same folder is allowed by ome standards

A `*.zarr` store on a local file system is just a directory/folder that contains the Zarr metadata. The Zarr and OME-NGFF specifications allows for any number of `*.zarr/`s in the same directory. Practically you can have as many `*.zarr/` folders in the same directory as the FS can handle.

Reply:
Perfect, thanks Ziwen. Would Zarr viewers know how to handle multiple .zarr stores? What would happen if I have the following file structure and I drag the `dataset` folder in a viewer like napari?
```
dataset
   FOV1
      data1.zarr
      data2.zarr
   FOV2
      data1.zarr
      data2.zarr
```